docs/source/happy.rst

```diff
@@ -300,6 +300,25 @@ Or exclude matching tests from being run:
    $ bin/test -t \!robot
 
 
+Running tests with the Google Chrome browser
+--------------------------------------------
+
+In our setup, robot framework uses Firefox to run the robot tests.
+If you want to use a different browser, you can define it in an environment variable and make sure, any necessary webdriver application are installed.
+The tests can be started like so::
+
+    ROBOT_BROWSER=NAME_OF_EXECUTABLE_TO_USE ./bin/test --all -m MODULE_TO_TEST
```

Comment:
IIRC its not the name of the executable. its a identifier mapped by selenium to the correct driver. So its has to be ``firefox``, ``chrome`` or ``phantomjs`` (and probably more, iirc theres also a remote driver....).